### PR TITLE
chore(background): Update websocket url

### PR DESCRIPTION
**Pull request by rylek90**
## :bug: Recommendations for testing
* Build button
* Log in
* Start time entry in webapp
* See if button fetches it

### CODE REVIEW

**Review by langri-sha — APPROVED** [review]
Small req, but otherwise LGTM :+1:

File: src/scripts/background.js [hunk 1]
```diff
@@ -320,7 +320,7 @@ window.TogglButton = {
     }
 
     try {
-      TogglButton.websocket.socket = new WebSocket('wss://stream.toggl.com/ws');
+      TogglButton.websocket.socket = new WebSocket('wss://track.toggl.com/stream');
```
[the remark this answers is not in the document]
Comment (rylek90) [reply src/scripts/background.js]: Thanks! Done :salute:

File: src/scripts/background.js [hunk 2]
```diff
@@ -320,7 +320,7 @@ window.TogglButton = {
     }
 
     try {
-      TogglButton.websocket.socket = new WebSocket('wss://stream.toggl.com/ws');
+      TogglButton.websocket.socket = Object.assign(new URL('/stream', process.env.API_URL), { protocol: 'wss:' }).href;
```
Comment (langri-sha) [inline src/scripts/background.js:323]:
```suggestion
      TogglButton.websocket.socket = new WebSocket(Object.assign(new URL('/stream', process.env.API_URL), { protocol: 'wss:' }).href);
```

**Review by langri-sha — CHANGES_REQUESTED** [review]

**Review by langri-sha — APPROVED** [review]
Yusss :raised_hands: ! LGTM :rocket:

**Pull request merged**